src/clusterfuzz/_internal/tests/core/bot/fuzzers/googlefuzztest/test_data/dummy_binary/no_op.c

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Google LLC
```

Comment:
Eh let's just mock new_process.run_and_wait or whatever is running the binary instead of using a fake binary.